[2023-06-14 19:59:26,308] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag_v3.second_task 2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14 19:59:26,322] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag_v3.second_task 2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14 19:59:26,324] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-06-14 19:59:26,326] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-06-14 19:59:26,328] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-06-14 19:59:26,347] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2023-06-13T00:00:00+00:00
[2023-06-14 19:59:26,353] {standard_task_runner.py:52} INFO - Started process 1379 to run task
[2023-06-14 19:59:26,359] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag_v3', 'second_task', '2023-06-13T00:00:00+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp4pn6pkl3', '--error-file', '/tmp/tmpteojcae7']
[2023-06-14 19:59:26,364] {standard_task_runner.py:77} INFO - Job 6: Subtask second_task
[2023-06-14 19:59:26,427] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag_v3.second_task 2023-06-13T00:00:00+00:00 [running]> on host c4a6634e48c9
[2023-06-14 19:59:26,477] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Servin
AIRFLOW_CTX_DAG_ID=first_dag_v3
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14 19:59:26,480] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-06-14 19:59:26,482] {bash.py:158} INFO - Running command: echo This is from the second task!
[2023-06-14 19:59:26,492] {bash.py:169} INFO - Output:
[2023-06-14 19:59:26,493] {bash.py:173} INFO - This is from the second task!
[2023-06-14 19:59:26,495] {bash.py:177} INFO - Command exited with return code 0
[2023-06-14 19:59:26,523] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag_v3, task_id=second_task, execution_date=20230613T000000, start_date=20230614T195926, end_date=20230614T195926
[2023-06-14 19:59:26,556] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-14 19:59:26,572] {local_task_job.py:146} INFO - Task exited with return code 0
